hnn_core/tests/test_viz.py

```diff
@@ -240,6 +247,96 @@ def test_dipole_visualization(setup_net):
     plt.close('all')
 
 
+class TestCellResponsePlotters:
+
+    @pytest.fixture(scope='class')
+    def class_setup_net(self):
```

Comment:
missing docstring

Reply:
@gtdang this comment is not addressed yet, otherwise I can go ahead and merge. Still see some test functions without a one-line docstring

Reply:
Ok, do you just want a doc string for these fixtures? Perhaps:  
"""Creates a base network from the default json for tests within this class"""
and 
"""Adds bursty drives with spikes to the base network for testing visualizations of spikes"""

Reply:
Yep that's good!